Upstream: No
Reason: GitVersion is a dependency that needs git history to return the current version.
Disable it and read from package project Version instead.

--- a/src/Recyclarr.Cli/Console/CliSetup.cs
+++ b/src/Recyclarr.Cli/Console/CliSetup.cs
@@ -2,7 +2,7 @@ using Autofac;
 using Recyclarr.Cli.Console.Commands;
 using Spectre.Console;
 using Spectre.Console.Cli;
-
+using System.Reflection;
 namespace Recyclarr.Cli.Console;
 
 internal static class CliSetup
@@ -89,7 +89,7 @@ internal static class CliSetup
 
             config.SetApplicationName("recyclarr");
             config.SetApplicationVersion(
-                $"v{GitVersionInformation.SemVer} ({GitVersionInformation.FullBuildMetaData})"
+                $"v{Assembly.GetEntryAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}"
             );
 
             AddCommands(config);

--- a/src/Recyclarr.Cli/Console/Setup/ProgramInformationDisplayTask.cs
+++ b/src/Recyclarr.Cli/Console/Setup/ProgramInformationDisplayTask.cs
@@ -1,13 +1,13 @@
 using Recyclarr.Cli.Console.Commands;
 using Recyclarr.Platform;
-
+using System.Reflection;
 namespace Recyclarr.Cli.Console.Setup;
 
 internal class ProgramInformationDisplayTask(ILogger log, IAppPaths paths) : IGlobalSetupTask
 {
     public void OnStart(BaseCommandSettings cmd)
     {
-        log.Debug("Recyclarr Version: {Version}", GitVersionInformation.InformationalVersion);
+        log.Debug("Recyclarr Version: {Version}", Assembly.GetEntryAssembly().GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion);
         log.Debug("App Data Dir: {AppData}", paths.ConfigDirectory);
     }
 
--- a/src/Recyclarr.Core/ResourceProviders/Infrastructure/ConfigTemplatesStrategy.cs
+++ b/src/Recyclarr.Core/ResourceProviders/Infrastructure/ConfigTemplatesStrategy.cs
@@ -13,7 +13,7 @@ public class ConfigTemplatesStrategy(ResourceRegistry<TemplateMetadata> registry
     public string Type => "config-templates";
 
     public IReadOnlyList<string> DefaultReferences =>
-        [$"v{GitVersionInformation.Major}", "master", "main"];
+        ["v8", "master", "main"];
 
     public IReadOnlyCollection<ResourceProvider> GetInitialProviders(
         IReadOnlyCollection<ResourceProvider> providers
